ext/crypto/00_crypto.js

```diff
@@ -717,6 +718,25 @@
             signature,
           }, data);
         }
+        case "ECDSA": {
+          // 1.
+          if (key[_type] !== "public") {
+            throw new DOMException(
+              "Key type not supported",
+              "InvalidAccessError",
+            );
+          }
```

Comment:
Can you exercise this code path with a test?

(Mea culpa, I may have forgotten to point that out in your other pull requests.)

Reply:
I am not 100% sure, but this is likely covered by WPT tests.

Reply:
The same thought occurred to me the moment I hit "Submit review". :-)

Reply:
Yup WPT covers this...https://github.com/denoland/wpt/blob/146f12e8df2cac6b1e60152145124a81dad60d38/WebCryptoAPI/sign_verify/ecdsa.js#L238 (using a "secret" key to sign ECDSA)
Adding WPT is pending

Reply:
uhhh so WPT depends on importKey. I've added a unit test for now